target/linux/mediatek/dts/mt7986a-asus-rt-ax59u.dts

```diff
@@ -230,17 +232,42 @@
 			partition@400000 {
 				label = "UBI_DEV";
 				reg = <0x400000 0x7c00000>;
+				compatible = "linux,ubi";
+
+				volumes {
+					ubi_factory: ubi-volume-factory {
+						volname = "Factory";
+					};
+				};
 			};
 		};
 	};
 };
 
+&ubi_factory {
```

Comment:
I know we used to do that a lot in OpenWrt but I have no idea why and I think recently we do less of such referencing.

If `ubi_factory` is in the same file (just few lines above) why not put `nvmem-layout` directly there? I think that would help reading DTS fle.

Reply:
I used to agree but was criticized for that and told that references are ok also if they reduce the level of indentation (which is why I applied this here). I don't mind either way, my screen is big enough and my text editor smart enough for me not to care.

Reply:
My 2 cents when the nvmem conversion was done all the reference were done as it was easier to handle them with a script. But the normal way would be that if the node is in the same dts then the node should be changed directly.